soe/java/DemoSOE/README.md

```diff
@@ -0,0 +1,158 @@
+Java based Service Orchestration Engine sample (SOE)
```

Comment:
Please do some formatting of this README. Headers should be delineated for each of the sections. You can refer to a markdown overview like this for details: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet